simple-weather/simple-weather/Model/CurrentWeather.swift

```diff
@@ -9,40 +9,39 @@
 import Foundation
 
 class CurrentWeather {
+    var cityName: String!
+    var date: String!
+    var weatherType: String!
+    var currentTemp: Double!
     
-    private var _cityName: String!
-    private var _date: String!
-    private var _weatherType: String!
-    private var _currentTemp: Double!
     
-    
-    var cityName: String {
-        if _cityName == nil {
-            _cityName = ""
-        }
-        return _cityName
-    }
-    
-    var date: String {
-        if _date == nil {
-            _date = ""
-        }
-        return _date
-    }
-    
-    var weatherType: String {
-        if _weatherType == nil {
-            _weatherType = ""
-        }
-        return _weatherType
-    }
-    
-    var currentTemp: Double {
-        if _currentTemp == nil {
-            _currentTemp = 0.0
-        }
-        return _currentTemp
-    }
+//    var cityName: String {
```

Comment:
Do not have any commented, or debug code (like setting label text as coordinates or printing coordinates) in Pull Request. Pull request must be completely clean, so it can be reviewed and instantly merged into develop.